public/script.js

```diff
@@ -3158,6 +3160,11 @@ class StreamingProcessor {
         this.abortController = new AbortController();
         this.firstMessageText = '...';
         this.timeStarted = timeStarted;
+        /**
+         * @type {number?}
+         */
```

Comment:
```suggestion
        /** @type {number?} */
```

Cut a line, save a tree

Reply:
Saved